src/autosuspend/checks/activity.py

```diff
@@ -94,23 +94,47 @@ def check(self):
 
 class Kodi(NetworkMixin, Activity):
 
-    QUERY = '?request={"jsonrpc": "2.0", "id": 1, ' \
-            '"method": "Player.GetActivePlayers"}'
+    @classmethod
+    def collect_init_args(cls, config) -> Dict[str, Any]:
+        try:
+            args = NetworkMixin.collect_init_args(config)
+            args['suspend_while_paused'] = \
+                config.getboolean('suspend_while_paused', fallback=False)
+            return args
+        except ValueError as error:
+            raise ConfigurationError(
+                'Configuration error ' + str(error)) from error
```

Comment:
Please use string formatting here.

Reply:
I copied this from `KodiIdleTime`, that's why it has no string formatting in the first place :) Fixed.

Reply:
Thanks. There are some things that have changed over time and that I should adapt at some point in time ;)